packages/catalog-search/package.json

```diff
@@ -60,7 +60,7 @@
     "react-test-renderer": "16.14.0"
   },
   "peerDependencies": {
-    "@edx/frontend-platform": "^1.9.6 || ^2.0.0",
+    "@edx/frontend-platform": "^3.6.0",
```

Comment:
Note: this is a breaking change to remove `v1` and `v2` support for `@edx/frontend-platform` from peer dependencies. Is there anything in `catalog-search` that precludes it from using `v1` or `v2` of `@edx/frontend-platform`?

If not, we should keep these peer dependencies in place, but expand them to include v3 of `@edx/frontend-platform` to avoid a breaking change for consumers. Also note, it should include the earliest compatible v3 version of `@edx/frontend-platform` (not just the latest version).

Similar feedback for the other packages in this repo.